core/src/main/java/nl/nn/adapterframework/doc/DocWriterNew.java

```diff
@@ -628,41 +696,103 @@ private void addExtraAttributesNotFromModel(XmlBuilder context, FrankElement fra
 		addClassNameAttribute(context, frankElement);
 	}
 
-	private void addConfigChildWithElementGroup(XmlBuilder context, ConfigChild child) {
+	private XmlBuilder addConfigChildWithElementGroup(XmlBuilder context, ConfigChild child) {
 		if(log.isTraceEnabled()) {
 			log.trace("Config child appears as element group reference");
 		}
-		ElementRole role = model.findElementRole(child);
-		defineElementTypeGroup(role);
-		DocWriterNewXmlUtils.addGroupRef(context, role.createXsdElementName(ELEMENT_GROUP), getMinOccurs(child), getMaxOccurs(child));
+		ConfigChildSet configChildSet = child.getOwningElement().getConfigChildSet(child.getSyntax1Name());
+		if(log.isTraceEnabled()) {
+			ThreadContext.push(String.format("Owning elenent [%s], ConfigChildSet [%s]", child.getOwningElement().getSimpleName(), configChildSet.toString()));
```

Comment:
Ik denk dat je hier een typfoutje gemaakt hebt. Er staat elenent in plaats van element.

Reply:
Opgelost.